tests/markdown.spec.ts

```diff
@@ -86,7 +86,12 @@ const cases = [
   {
     title: 'render ordered lists',
     markdown: '1. option-1\n2. option-2',
-    html: '<ol><li>option-1</li><li>option-2</li></ol>',
+    html: '<ol start="1"><li>option-1</li><li>option-2</li></ol>',
```

Comment:
we can also keep the old behavior as well. only add start attr if required otherwise default. Will there be any issue with that behavior?

Reply:
I've restored the original behavior so that it doesn't add the attribute when starting with `1.`